plasmapy/formulary/ionization.py

```diff
@@ -0,0 +1,55 @@
+""" This module gathers functions relating to ionization states and the properties thereof.
+"""
+__all__ = ["Z_bal"]
+
+from astropy import units as u
+from plasmapy.utils.decorators import (
+    angular_freq_to_hz,
+    check_relativistic,
+    validate_quantities,
+)
+from astropy.constants import c, a0, k_B
+from numpy import pi, exp, sqrt, log
+
+
+def Z_bal(n: u.m ** -3, T_e: u.K):
+    r"""
+    Z_bal is derived from the Saha equation with the assumptions that
```

Comment:
Likewise, here we need to describe what it is this calculates, and only then can we state the assumptions that go into it.